unity-client/Assets/UnityGLTF/Scripts/GLTFSceneImporter.cs

```diff
@@ -2126,6 +2126,18 @@ protected IEnumerator ConstructUnityMesh(MeshConstructionData meshConstructionDa
                 mesh.RecalculateNormals();
             }
 
+            if (ShouldYieldOnTimeout())
+            {
+                yield return YieldOnTimeout();
+            }
+
+            mesh.Optimize();
```

Comment:
I'm not sure how this may impact performance when done in runtime, if that hits performance somehow, maybe we should check somehow if this is running in the world or at "ab-conversion-time" ?

Reply:
Just changed it to take place only in editor time until we can benchmark it better.